tools/wakeuptime.py

```diff
@@ -195,30 +180,43 @@ def signal_ignore(signal, frame):
 
     if not folded:
         print()
+    missing_stacks = 0
+    has_enomem = False
     counts = b.get_table("counts")
+    stack_traces = b.get_table("stack_traces")
     for k, v in sorted(counts.items(), key=lambda counts: counts[1].value):
+        # handle get_stackid errors
+        # check for an ENOMEM error
+        if k.w_k_stack_id == -errno.ENOMEM:
+            missing_stacks += 1
+            continue
```

Comment:
Another missing stacks will be the one being silently replaced ones and there are no ways to counting them right now. So I guess we can ignore them and your counting with -ENOMEM is the only option we can do now.